### PR TITLE
make links clickable

### PR DESCRIPTION
**Pull request by yetkinergun**

### CODE REVIEW

**Comment by omonk** [comment]
I think dist/ should be in .gitignore, we don't want compiled files in the repo

**Comment by wmonk** [comment]
`dist` should be included, it's the way it's install into the live chat repo. Although at some point we should make it a package in this repo.

File: src/ChatBubble/index.tsx [hunk 1]
```diff
@@ -77,9 +78,11 @@ export default function ChatBubble(props: ChatBubbleProps) {
             {props.senderName}
           </BubbleGroupHeader>
         )}
-        <MessageUI me={props.message.fromMe} data-test-id="rcu-bubble-text">
-          {props.message.message}
-        </MessageUI>
+        <Linkify>
```
Comment (wmonk) [inline src/ChatBubble/index.tsx:84]: Can we make this target blank?

Comment (yetkinergun) [reply src/ChatBubble/index.tsx:84]: its blank by default

**Review by wmonk — APPROVED** [review]

**Pull request merged**